src/Api/Clips.php

```diff
@@ -68,6 +69,17 @@ public function getTopClips($channel = null, $cursor = null, $game = null, $limi
                 throw new TwitchApiException(sprintf('Only a maximum of 10 games can be queried. %d requested.', $count));
             }
         }
+        
+        if ($language) {
+            if (!is_string($language)) {
+                throw new InvalidTypeException('language', 'string', gettype($language));
+            }
+
+            $language = trim($language, ', ');
+            if ($count = count(explode(',', $language)) > 28) {
```

Comment:
```suggestion
            if (($count = count(explode(',', $language))) > 28) {
```

Otherwise `$count` will be a boolean instead of an integer.

Reply:
To be honest this entire code base is bad, that's on me :laughing: